src/code/code_800EC960.c

```diff
@@ -1751,710 +1771,715 @@ void func_800EEA50(GfxPrint* printer) {
 
             SETCOL(255, 255, 255);
             GfxPrint_SetPos(printer, 3, 7);
-            GfxPrint_Printf(printer, "SE PR  : %02x", D_8013331C[D_80131F20[0]][D_80131F20[1]].unk_0);
+            GfxPrint_Printf(printer, "SE PR  : %02x",
+                            D_8013331C[sAudioSEParamChgWork[0]][sAudioSEParamChgWork[1]].unk_0);
             break;
 
-        case 14:
+        case PAGE_FREE_AREA:
             GfxPrint_SetPos(printer, 3, 4);
             SETCOL(255, 255, 255);
-            GfxPrint_Printf(printer, "env_fx %d code_fx %d SPEC %d", D_80130614, D_80130618, D_80133414);
+            GfxPrint_Printf(printer, "env_fx %d code_fx %d SPEC %d", sAudioEnvReverb, sAudioCodeReverb, gAudioSpecId);
 
-            if (sAudioUpdateTaskEnd == sAudioUpdateTaskStart) {
-                D_80131C90 = (f32)((sAudioUpdateEndTime - sAudioUpdateStartTime) * 64ULL / 3) / 50000000.0f;
-                if (D_80131C94 < D_80131C90) {
-                    D_80131C94 = D_80131C90;
+            if (sAudioUpdateTaskStart == sAudioUpdateTaskEnd) {
+                sAudioUpdateDuration = (f32)((sAudioUpdateEndTime - sAudioUpdateStartTime) * 64ULL / 3) / 50000000.0f;
+                if (sAudioUpdateDurationMax < sAudioUpdateDuration) {
+                    sAudioUpdateDurationMax = sAudioUpdateDuration;
                 }
             }
 
             GfxPrint_SetPos(printer, 3, 6);
-            GfxPrint_Printf(printer, "SOUND GAME FRAME NOW %f", D_80131C90);
+            GfxPrint_Printf(printer, "SOUND GAME FRAME NOW %f", sAudioUpdateDuration);
 
             GfxPrint_SetPos(printer, 3, 7);
-            GfxPrint_Printf(printer, "SOUND GAME FRAME MAX %f", D_80131C94);
+            GfxPrint_Printf(printer, "SOUND GAME FRAME MAX %f", sAudioUpdateDurationMax);
 
             GfxPrint_SetPos(printer, 3, 9);
             GfxPrint_Printf(printer, "SWITCH BGM MODE %d %d %d (FLAG %d)", D_8013061C, D_80130634, D_80130638,
                             D_80130654);
 
             GfxPrint_SetPos(printer, 3, 10);
-            GfxPrint_Printf(printer, "ENEMY DIST %f VOL %3d", D_80130620, D_80130624);
+            GfxPrint_Printf(printer, "ENEMY DIST %f VOL %3d", sAudioEnemyDist, sAudioEnemyVol);
 
             GfxPrint_SetPos(printer, 3, 11);
-            GfxPrint_Printf(printer, "GANON DIST VOL %3d", D_8016B8B4);
+            GfxPrint_Printf(printer, "GANON DIST VOL %3d", sAudioGanonDistVol);
 
             GfxPrint_SetPos(printer, 3, 12);
-            GfxPrint_Printf(printer, "DEMO FLAG %d", D_8013060C);
+            GfxPrint_Printf(printer, "DEMO FLAG %d", sAudioDemoActive);
 
             GfxPrint_SetPos(printer, 3, 12);
-            if (D_8016BAA8 == 1) {
-                GfxPrint_Printf(printer, "MARON BGM DIST %f", D_8016BAAC);
-                D_8016BAA8 = 0;
+            if (sAudioHasMaronBgm == true) {
+                GfxPrint_Printf(printer, "MARON BGM DIST %f", sAudioMaronBgmDist);
+                sAudioHasMaronBgm = false;
```

Comment:
ah I see, these names are coming from stings lol
In this case Maron would be Malon. But I think we should probably try to avoid just blindly using the strings, and name them on our own terms. But yeah if we want to do this in another pass id be fine with it

Reply:
Oh, haha, shows how well I know the game for not noticing. I think in this case it's probably a decent name with the misspelling fixed. But yeah, none of the symbol of function names here are final, this code will need several more doc passes.